### PR TITLE
Development quickstart

### PR DESCRIPTION
**Pull request by di**
Here's what I had to do to get things running locally.

### CODE REVIEW

File: pytx/settings/development.py [hunk 1]
```diff
@@ -3,7 +3,7 @@
 
 from pytx.settings.base import *
 
-SECURE_SSL_REDIRECT = True
+SECURE_SSL_REDIRECT = False
```
Comment (pizzapanther) [inline pytx/settings/development.py:6]: I use SSL on dev to test the service worker but I'll make a local.py so we can override individually

File: requirements-dev.txt [hunk 2]
```diff
@@ -1,2 +1,3 @@
+-r requirements.txt
```
Comment (pizzapanther) [inline requirements-dev.txt:1]: cool didn't know that trick

File: pytx/settings/development.py [hunk 3]
```diff
@@ -19,3 +19,7 @@
 MEDIA_URL = STATIC_URL + 'uploads/'
 
 STATICFILES_DIRS.append(MEDIA_ROOT)
+
+SECRET_KEY = "a insecure development secret"
```
Comment (pizzapanther) [inline pytx/settings/development.py]: remove this and make a `.env` instead. `.env` will be loaded automatically

should look like:

```
SECRET_KEY=narf
```

**Review by pizzapanther — CHANGES_REQUESTED** [review]

File: pytx/settings/development.py (hunk 1 above)
Comment (di) [reply pytx/settings/development.py:6]: 👍 feel free to push to my branch if you'd like to do it in this PR.

File: README.md [hunk 4]
```diff
@@ -1 +1,40 @@
...
\ No newline at end of file
+# PyTexasBackend
+
+## Development Quickstart
+
+Clone the repo:
+
+```
+git clone git@github.com:pytexas/PyTexasBackend.git
+git submodule init
```
Comment (pizzapanther) [inline README.md]: @di just fyi so you pull down the submodules

File: README.md [hunk 5]
```diff
@@ -1 +1,40 @@
...
+git submodule update
```
Comment (di) [inline README.md]: These two lines can also just be `git submodule update --init`

File: README.md [hunk 6]
```diff
@@ -1 +1,55 @@
...
+$ python manage.py migrate
+```
+
+Run the development server:
+```
+# backend
+python manage.py runserver
+
+# frontend
+gulp watch
```
Comment (pizzapanther) [inline README.md:44]: @di one more FYI to build JS frontend

**Review by pizzapanther — APPROVED** [review]

**Pull request merged**